README.rst

```diff
@@ -0,0 +1,27 @@
+Equistore
+=========
+
+|test| |docs|
+
+Equistore is a specialized data storage format suited to all your
+atomistic simulation needs and more.
+
+The core of the library is written in Rust and we provide
+APIs for C/C++ and Python as well.
+
+.. warning::
+
+    **Rascaline is still as the proof of concept stage. You should not use it for
+    anything important.**
```

Comment:
Wrong project =) And I think we can omit this altogether

Reply:
Done.